Packs/qualys/Integrations/Qualysv2/Qualysv2.py

```diff
@@ -793,7 +793,7 @@
         "resp_type": "text",
     },
     "qualys-host-list-detection": {
-        "api_route": API_SUFFIX + "asset/host/vm/detection/?action=list",
+        "api_route": API_SUFFIX + "asset/host/vm/detection/?action=list&show_qds=1&show_qds_factors=1",
```

Comment:
lets add a quick comment about this change.

Reply:
Added comment